### DISCUSSION TITLE
Why it take so long time ?

### DISCUSSION THREAD

**Discussion by mshmoon** [Universite-Gustave-Eiffel/NoiseModelling#561] [Q&A]
Why does it take a lot of time for me to process the downloaded osm.pbf file using noisemodelling. At this processing speed, I can't finish processing for several hours, but this file only has  14M bytes.

![image](https://user-images.githubusercontent.com/38131350/228723753-39fdc468-a09b-47a7-a029-70abccdcc9f7.png)

**Comment by mshmoon** [comment]
![image](https://user-images.githubusercontent.com/38131350/228729213-b93128fc-f1ab-4530-b04b-b467b71c3133.png)

**Reply by pierromond** [reply]
Hi @mshmoon...

We can't really help you... Which version of NM are you using ? We can see than it took 1 minute to read the osmpbf file... not so coherent with your message. Excuse me but your post is quite unclear. Can you post your log file ?
